bitnami/redis/templates/redis-master-statefulset.yaml

```
@@ -68,7 +68,7 @@ spec:
       schedulerName: "{{ .Values.master.schedulerName }}"
       {{- end }}
       containers:
-      - name: {{ template "redis.fullname" . }}
+      - name: {{ template "redis.Name" . }}
```

Please, use the helper name as defined (`name` lowercase):

https://github.com/bitnami/charts/blob/556b6028e199626d728366359526342fd4621e49/bitnami/redis/templates/_helpers.tpl#L5
```suggestion
      - name: {{ template "redis.name" . }}
```